src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="timesheet">
+  <div class="timesheet__header">
+    <div class="timesheet__header__container">
+      <a title="Retour à l'accueil" [routerLink]="['']">
+        <img class="timesheet__header__container__logo" src="assets/images/logo.white.svg" alt="logo acrabadabra"
+          (error)="$event.target.removeAttribute('onerror'); $event.target.src = 'assets/images/logo.white.png';">
+      </a>
+    </div>
+  </div>
+  <router-outlet></router-outlet>
```

Comment:
You have a `router-outlet` so this Component cannot be named `header` 🤔 
It's a layout or something like this.